geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ConnectCommand.java

```diff
@@ -152,7 +154,7 @@ public Result connect(
       if (skipSslValidation) {
         HttpsURLConnection.setDefaultHostnameVerifier((String s, SSLSession sslSession) -> true);
       }
-      result = httpConnect(gfProperties, url);
+      result = httpConnect(gfProperties, url, skipSslValidation);
```

Comment:
the flag is passed down and we can continue to pass the flag down to getTrustManager() instead of setting it to SSLConfig.